examples/large_models/inferentia2/util/inf2_save_split_checkpoints.py

```diff
@@ -40,6 +41,12 @@ def opt_amp_callback(model: OPTForCausalLM, dtype: torch.dtype) -> None:
     default="./model-splits",
     help="Output directory for downloaded model files",
 )
+subparsers = parser.add_subparsers(dest="subparser")
+parser_neuron_cache = subparsers.add_parser("generate_neuron_cache")
```

Comment:
these don't feel like they should be required?

Reply:
I've included the `generate_neuron_cache` as an optional step when running `inf2_save_split_checkpoints` so that the following arguments such as `neuron_cache_dir`, `batch_size` etc.. are required only if `generate_neuron_cache` is specified. So, we require only the model checkpoint we can just run `python ../util/inf2_save_split_checkpoints.py --model_name meta-llama/Llama-2-13b-hf --save_path './llama-2-13b-split'
` as before.